openshift_images/configuring-samples-operator.adoc

```diff
@@ -19,3 +19,25 @@ include::modules/samples-operator-overview.adoc[leveloffset=+1]
 include::modules/samples-operator-configuration.adoc[leveloffset=+1]
 
 include::modules/samples-operator-crd.adoc[leveloffset=+1]
+
+[id="samples-operator-alt-registry"]
+== Using the Samples Operator with an alternate registry
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
+
+[id="samples-preparing-bastion"]
+=== Preparing the bastion host
```

Comment:
Trevor has a PR open to change "bastion" to "mirror" here: https://github.com/openshift/openshift-docs/pull/19435/files

If your change is ready to merge, then we can open a follow-up after his merges.